### PR TITLE
(#184) create new message content table

### PR DESCRIPTION
**Pull request by Pixailz**
concerning only the chat dm because chat channel has refactor in progress

- [x] player can send string or game-invite type
- [x] player can send string and game-invite type

### CODE REVIEW

**Comment by Pixailz** [comment]
can now send string and game-invite at the same time

image

to do so:
1. Write a string
2. Click on the '+' button
3. Click on GameInvite
4. Write the room id (or something else)
5. send it

the form is reseted after sending a message

**Comment by Pixailz** [comment]
- fix several jerk code that was causing a lot of trouble
- merge create custom game with the lobby
image
- send notifications and message on game invite
  - integrate a button into the chat to join a game, with a default string
image
- make chad private game proof against the matchmaking

**Comment by Pixailz** [comment]
need to be tested, but ready to PR :)

File: src/nestjs/src/modules/database/database.module.ts [hunk 1]
```diff
@@ -20,6 +22,8 @@ import { DBFriendService } from "./friend/service";
 import { DBFriendRequestService } from "./friendRequest/service";
 import { DBBlockedService } from "./blocked/service";
 import { DBNotificationService } from "./notification/service";
+import { DBPlayerScoreService } from "./game/player-score/service";
+import { DBMessageContentService } from "./messageContent/service";
```
Comment (clinche) [inline src/nestjs/src/modules/database/database.module.ts:26]: duplicate of lines 38-39

**Review by clinche — CHANGES_REQUESTED** [review]
nestjs wont launch due to duplicates and angular fails with a lot of errors, the first being
```
Error: src/app/app.component.html:1:1 - error NG8001: 'router-outlet' is not a known element:
1. If 'router-outlet' is an Angular component, then verify that it is part of this module.
2. If 'router-outlet' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message.
```

**Comment by Pixailz** [comment]
fixed :)

**Review by clinche — APPROVED** [review]
looks noice to me

**Pull request merged**